### ISSUE TITLE
Link to actual TBS definition of "open standard".

### ISSUE THREAD

**Issue by kfogel** [canada-ca/open-source-logiciel-libre#63]
In the [Guide for open standards](https://github.com/canada-ca/open-source-logiciel-libre/blob/master/en/guides/open-standards.md), it would be good to link directly to the TBS definition of "open standard".

I tried to chase down the definition, in the various references given, but was unable to.  I was looking for some key phrases like "royalty-free", etc.

Unlike the term "open source", there isn't a completely clear, widely agreed-on definition of "open standard" (although the [definition of "open standard" given by the Open Source Initiative](https://opensource.org/osr) is good and is maybe the closest thing we have to a formal definition of the term -- see also [additional commentary re "FRAND" here](https://opensource.org/node/616)).  Thus the importance both of linking to the TBS definition, if there is one, and of ensuring that that definition is a solid one that allows unrestricted, non-monopolized, royalty-free implementation in open source software.

**Comment by gcharest** [comment]
You are correct, the Open Standard section has not yet been implemented. Very limited work has been done on that topic (although I know it's a very important one).

Part of the issue lies with the fact that it is not as easily or widely defined (or rather accepted) as part of our research. OSS best practices on the other end are widely defined and practiced so it was in some ways a lower hanging fruit.
